geoportal/tests/functional/test_dbreflection.py

```diff
@@ -27,6 +27,8 @@
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
+# pylint: disable=C0111,W0201,W0212
```

Comment:
I find the human readable name easier to understand, example : # pylint: disable=no-self-use

Reply:
If we need it in all files, it might be possible to add it in config file or command line.

Reply:
Done :-)

Reply:
As I know it's not easily possible in the IDE, because I want it only for the test files...